cc_licenses/templates/includes/deed_40_body.html

```diff
@@ -91,12 +92,4 @@ <h3 class="b-header has-text-black padding-bottom-big padding-top-normal" style=
   </p>
 </div>
 
-<style>
-  #deed-body {
-    background-color: rgb(255, 255, 255);
-    border-top: 10px solid rgb(60, 92, 153);
-    border-bottom: 5px solid rgb(176, 176, 176);
-    border-left: 5px solid rgb(176, 176, 176);
-    border-right: 5px solid rgb(176, 176, 176);
-  }
-</style>
+xxx-xxx
```

Comment:
Hmm, what will this represent? Seems like this addition is not shown in the wireframes? Is this from a slack discussion?

Reply:
Oops, didn't mean to leave this in after debugging the layout, will remove it.  (The CSS moved to the common deed page template.)